test/quantization/test_quantize_script.py

```diff
@@ -1794,6 +1810,39 @@ def forward(self, x):
         FileCheck().check("quantized::linear_dynamic") \
                    .run(model.graph)
 
+    def test_dynamic_qconfig_w_types(self):
```

Comment:
same for this one, we can have end to end tests in some other places

Reply:
also there is probably nothing special for dynamic regarding this function, so you don't need to replicate this test for dynamic, unless you are doing integration test